create-code-deploy-application.sh

```diff
@@ -2,18 +2,9 @@
 
 APPLICATION_NAME=$1
 DEPLOYMENT_GROUP_NAME=$2
-INSTANCE_PROFILE_NAME=$3
-BUCKET_NAME=$4
+BUCKET_NAME=$3
 
-
-fetchInstanceProfile(){
-    CODE_DEPLOYER_PROFILE_ARN=$(aws iam get-instance-profile \
-                                --instance-profile-name $INSTANCE_PROFILE_NAME \
-                                --query InstanceProfile.Roles[0].Arn \
-                                --output text)
-
-    echo "$CODE_DEPLOYER_PROFILE_ARN"
-}
+CODE_DEPLOYER_ROLE_ARN=arn:aws:iam::022990953738:role/code-deploy-role
```

Comment:
discussed IRL. the ARN is entirely based on the role name so this isn't a big deal.